manual/core/integration/README.md

```diff
@@ -496,6 +497,18 @@ don't use geospatial types anywhere in your application, you can exclude the dep
 </dependency>
 ```
 
+Starting with driver 4.14.0 Esri has been changed to an optional dependency.  You no longer have to
+explicitly exclude the dependency if it's not used, but if you do wish to make use of the Esri
+library you must now explicitly specify it as a dependency :
+
+```xml
+<dependency>
+  <groupId>com.esri.geometry</groupId>
+  <artifactId>esri-geometry-api</artifactId>
+  <version>${esri.version}</version>
```

Comment:
We could add a note here about the fact that the driver requires Esri 1.2.x, wdyt? This could be useful for developers since it's such an old version.

Reply:
I completely agree with this.  My original version of these docs spelled out the version of ESRI to include but I shifted over the the prop-based model in use here to avoid customer trying to use a newer version.  That said, I readily agree that an additional warning here about the older version would be worthwhile... I'll add that in.

Reply:
Added the trailing note here re: the use of an older ESRI version.  Does that address your concerns here?

Reply:
We are good thanks!